newIDE/app/src/EffectsList/index.js

```diff
@@ -86,6 +87,233 @@ export const useEffectOverridingAlertDialog = () => {
   };
 };
 
+const Effect = ({
+  layerRenderingType,
+  target,
+  project,
+  resourceManagementProps,
+  effectsContainer,
+  effect,
+  effectRef,
+  removeEffect,
+  copyEffect,
+  pasteEffectsBefore,
+  chooseEffectType,
+  allEffectMetadata,
+  onEffectsUpdated,
+  onEffectsRenamed,
+  nameErrors,
+  setNameErrors,
+  connectDragSource,
+}: {|
+  layerRenderingType: '2d' | '3d',
+  target: 'object' | 'layer',
+  project: gdProject,
+  resourceManagementProps: ResourceManagementProps,
+  effectsContainer: gdEffectsContainer,
+  effect: gdEffect,
+  effectRef: {| current: ?any |} | null,
```

Comment:
I think you'd rather forward the ref rather than setting it with the props.
But if it's too much of a change, it's not that important